core/config/v2/validate.go

```diff
@@ -191,3 +191,11 @@ func (u UniqueStrings) isDupe(s string) bool {
 	}
 	return ok
 }
+
+type ErrOverride struct {
+	Name string
+}
+
+func (e ErrOverride) Error() string {
+	return fmt.Sprintf("%s: override: %s", e.Name, "overrides (duplicate keys or list elements) are not allowed for multiple secrets files")
```

Comment:
Let's simplify the const part:
```suggestion
	return fmt.Sprintf("%s: override: overrides (duplicate keys or list elements) are not allowed for multiple secrets files", e.Name)
```
And drop the prefix? Seems unnecessary, no?
```suggestion
	return fmt.Sprintf("%s: overrides (duplicate keys or list elements) are not allowed for multiple secrets files", e.Name)
```

Reply:
Nice catch, will do!